server/app.py

```diff
@@ -1,21 +1,48 @@
+import json
+import os
 from flask import Flask
+from flask_restful import Api
+from flask_sqlalchemy import SQLAlchemy
+import config
+
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
 
-import json
-import os
-
 # Configuration
 GOOGLE_CLIENT_ID = os.environ.get("GOOGLE_CLIENT_ID", None)
 GOOGLE_CLIENT_SECRET = os.environ.get("GOOGLE_CLIENT_SECRET", None)
 GOOGLE_DISCOVERY_URL = (
     "https://accounts.google.com/.well-known/openid-configuration"
 )
+flask_app = Flask(__name__)
+crm_api = Api(flask_app)
+
+# Add to database
+flask_app.config['SQLALCHEMY_DATABASE_URI'] = config.SQLALCHEMY_DATABASE_URI
+flask_app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+# Flask Sessions key. See https://flask.palletsprojects.com/en/1.1.x/quickstart/#sessions
+flask_app.config['SECRET_KEY'] = 'some-secret-string'
```

Comment:
probably put some-secret-string in your .env file 
Now we should probably gitignore the .env file

Reply:
will do!